tests/unit/backend/corpora/common/entities/test_project.py

```diff
@@ -0,0 +1,38 @@
+import os
+import sys
+import unittest
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), "..", "..", "..", "..", "..", ".."))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+from backend.corpora.common.entities.project import Project
+from backend.corpora.common.utils.exceptions import CorporaException
+
+
+class TestProject(unittest.TestCase):
+    def setUp(self):
+        self.uuid = "f753497f-f8a5-48b5-b344-de9ad5a4354d"
+        self.status = "LIVE"
```

Comment:
This is a constant that probably should be in CorporaConstants? Or somewhere else? Best to not hardcode these constants because one day when we change the value, it'll be a nightmare to try and refactor it everywhere.

Reply:
This test file is now deprecated. Several constants defined in the ORM are defined as Enums in `corpora_orm.py`. I noticed you use `namedtuple`s as alternative to this, and they seem to be more lightweight/simpler usage. I would be open to moving the ORM Enum constants to `namedtuple`s `CorporaConstants` in separate refactor PR. Thoughts?

Reply:
I think enum is the more pythonic way of tracking differences in state. Just import the Enum from corpora_orm.py.

Reply:
Yep, I think @Bento007 's suggestion makes sense. The enums are specific to the database so it makes sense to me to import it directly from the ORM.

Reply:
Done